src/pages/Detail/CommonDetail/ProductContentBottom.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import * as P from './ProductDetailStyle';
+import { BiPlusMedical } from 'react-icons/bi';
+import { ImMinus } from 'react-icons/im';
+
+export default function ProductContentBottom({ detailContent }) {
+  const [count, setCount] = useState(1);
+
+  const handleQuantity = type => {
+    if (type === 'plus') {
+      setCount(count + 1);
+    } else if (type === 'minus') {
+      if (count === 1) {
+        return;
+      }
+      setCount(count - 1);
+    }
+  };
```

Comment:
```suggestion
  const handleQuantity = value => {
    if (count === 1 && value === -1) return null;
    setCount(prev => prev + value);
  };
```

수량을 변경하는 함수를 구현할 수 있겠습니다.
value값으로는 plus일때는 1을 인자로 넘겨주고, minus일때는 -1을 인자로 넘겨주는 방식으로 구현합니다.